package.json

```diff
@@ -33,8 +33,8 @@
   "dependencies": {
     "@angular-devkit/schematics": "8.3.22",
     "@schematics/angular": "8.3.22",
-    "azure-function-express": "2.0.0",
     "cors": "2.8.5",
+    "reflect-metadata": "^0.1.13",
```

Comment:
`reflect-metadata` should not be defined as a dependency (multiple copies of the `reflect-metadata` create side-effects)

Reply:
Would it be ok to move it to peerDepenencies then? as when I linked the lib to debug it, it was complaining about `recflect-metadata` being missing, that's why I added it.

Reply:
@sinediad `peerDependencies` is fine

Reply:
Done